### PR TITLE
Dzollo/release updates v2.3

### PR DESCRIPTION
**Pull request by denniszollo**
Update changelog and readme
filter out unknown sbp message warning for release console

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,439 @@
 # Change Log
 
+## [v2.2.0-develop-2019061023](https://github.com/swift-nav/piksi_tools/tree/v2.2.0-develop-2019061023) (2019-06-10)
```
Comment (silverjam) [inline CHANGELOG.md]: v2.3.17?

File: piksi_tools/console/console.py [hunk 2]
```diff
@@ -56,6 +55,8 @@
                                        mode_dict, resource_filename,
                                        icon, swift_path, DR_MODE, DIFFERENTIAL_MODES)
 
+warnings.filterwarnings("ignore", ".*No message found for msg_type.")
```
Comment (silverjam) [inline piksi_tools/console/console.py]: `git merge origin/master` into this branch should drop this change

File: CHANGELOG.md (hunk 1 above)
Comment (silverjam) [reply CHANGELOG.md]: Never mind, I guess this changelog script pulls in all of the dev branches too?

Comment (denniszollo) [reply CHANGELOG.md]: yes but I'll manually delete this top part as it is confusing for what we are trying to document

**Comment by silverjam** [comment]
Note that there's also this one: https://github.com/swift-nav/piksi_tools/pull/1023

**Pull request merged**